avocado/core/job.py

```diff
@@ -371,31 +370,15 @@ def _log_avocado_datadir():
         job_log.info('logs     ' + data_dir.get_logs_dir())
         job_log.info('')
 
-    def _log_variants_tree(self, variant):
```

Comment:
It was not a matter of difficulty of squashing commits.  Given the changes that were requested (which were pretty minor), it was a matter of changing the commits in a way that the review of the next version would be more difficult.

Again, the changes in a commit have to stand for themselves in the commit itself, and *also* in the context of the PR.